ios/Classes/ComAppceleratorUrlSessionModule.m

```diff
@@ -128,17 +128,34 @@ -(id)addBackgroundUploadTask:(id)args
     
     ComAppceleratorUrlSessionSessionProxy *session = nil;
     NSString *url = nil;
-    TiBlob *blob = nil;
+    NSString *fileUrlString = nil;
+    NSString *method = nil;
+    NSURL *fileURL = nil;
+    NSDictionary *headers = nil;
     
     ENSURE_ARG_FOR_KEY(session, args, @"session", ComAppceleratorUrlSessionSessionProxy);
     ENSURE_ARG_FOR_KEY(url, args, @"url", NSString);
-    ENSURE_ARG_FOR_KEY(blob, args, @"data", TiBlob);
+    ENSURE_ARG_FOR_KEY(fileUrlString, args, @"file", NSString);
```

Comment:
Sounds good. For background uploads with the app in suspended mode, we do need to use the `NSURLSessionUploadTask *task = [[session session] uploadTaskWithRequest:request fromFile:fileURL];` selector. So I'll handle both cases.